rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayNode.scala

```diff
@@ -884,19 +885,29 @@ object DisplayNode extends Loggable {
 
   private[this] def removeNode(nodeId: NodeId) : JsCmd = {
     val modId = ModificationId(uuidGen.newUuid)
+    import com.normation.rudder.services.servers.DeletionResult._
     removeNodeService.removeNode(nodeId, modId, CurrentUser.getActor) match {
-      case Full(entry) =>
+      case Full(Success) =>
         asyncDeploymentAgent ! AutomaticStartDeployment(modId, CurrentUser.getActor)
         onSuccess
-
+      case Full(PostHookFailed(postHook)) =>
+        logger.error(postHook.msg)
+        onFailure(nodeId, "Node was correctly deleted, but there was an error while running post-deletion hooks", postHook.msg)
+      case Full(PreHookFailed(int : Interrupt)) =>
+        val message = "Node deletion was interrupted because one the pre-deletion hooks exited with interrupt code (100)"
```

Comment:
`because one *of* the pre-deletion` 
Do we have the hook name? It will make sense to add it here.